salt/utils/async.py

```diff
@@ -94,10 +94,10 @@ def __del__(self):
                 # their associated io_loop is closed to allow for proper
                 # cleanup.
                 self.async.close()
-            self.io_loop.close()
             # Other things should be deallocated after the io_loop closes.
             # See Issue #26889.
```

Comment:
This comment no longer seems to apply, since you moved `self.io_loop.close()`. So if it no longer applies, you should remove the comment as well.

Reply:
Removed. Thanks.